### PR TITLE
Record dongle events in the PostgreSQL database in the teleserver

### PR DESCRIPTION
**Pull request by v2xsami**
Record dongle events in the PostgreSQL database in the teleserver.

JIRA=VE-73

### CODE REVIEW

File: firmware_v5/telelogger/telelogger.ino [hunk 1]
```diff
@@ -614,7 +614,7 @@ bool initialize()
     }
     else
     {
-      NotifyDongleEvent(EVENT_LOGIN);
+      NotifyDongleEvent("wakeup");
```
Comment (shumail92) [inline firmware_v5/telelogger/telelogger.ino]: Preferably, there should be a constant for `"wakeup"` like `EVENT_WAKEUP`.

File: firmware_v5/telelogger/telelogger.ino [hunk 2]
```diff
@@ -734,7 +734,7 @@ void standby()
 {
   if (state.check(STATE_CONNECTED))
   {
-    NotifyDongleEvent(EVENT_LOGOUT);
+    NotifyDongleEvent("standby");
```
Comment (shumail92) [inline firmware_v5/telelogger/telelogger.ino]: Same as above comment. `"standby"` should be used as a constant `EVENT_STANDBY`.

File: server/teleserver/main.cpp [hunk 3]
```diff
@@ -70,15 +71,22 @@ int MainLoop(int argc,char* argv[])
...
 			break;
 		}
 
 	}
 
 	const v2x::milliseconds tick_interval(100);
 
-	v2x::TeleServer server;
+	v2x::PostgreSqlJournal::Database db_config;
+	db_config.host = "127.0.0.1";
```
Comment (shumail92) [inline server/teleserver/main.cpp]: Please move the db configuration (host, port user, password, dbName) out into some config file so configuration can be easily changed.

File: server/teleserver/teleserver.cpp [hunk 4]
```diff
@@ -5,12 +5,45 @@
...
+    const auto& msg = packet.message.message;
+    const bool is_valid_msg = (msg == "wakeup" ||
+        msg == "standby" || msg == "data");
+    if (!is_valid_msg)
+    {
+        WARN("Ignoring '%1' message from %2:%3", msg, packet.ip_addr, packet.ip_port);
+        return;
+    }
+
+    if (msg == "wakeup" || msg == "standby")
```
Comment (shumail92) [inline server/teleserver/teleserver.cpp]: If you change and use constants in `telelogger.ino`, please try to reuse those constants here.

Comment (v2xsami) [reply server/teleserver/teleserver.cpp]: @shumail92 done

File: firmware_v5/telelogger/telelogger.ino (hunk 1 above)
Comment (v2xsami) [reply firmware_v5/telelogger/telelogger.ino]: @shumail92 done

File: firmware_v5/telelogger/telelogger.ino (hunk 2 above)
Comment (v2xsami) [reply firmware_v5/telelogger/telelogger.ino]: @shumail92 done

File: server/teleserver/main.cpp (hunk 3 above)
Comment (v2xsami) [reply server/teleserver/main.cpp]: @shumail92 I'd like to do the soft configuration in a separate patch for all the parameters of the server if that's ok

Comment (shumail92) [reply server/teleserver/main.cpp]: @v2xsami Sure.

**Pull request merged**